templates/data/incubator-buildpack-data.yaml

```diff
@@ -344,6 +348,52 @@ templates:
                   local:
                     - extra_refs_kyma_cli
                     - reconciler_publish_pr_cli_jobConfig
+              - jobConfig:
+                  name: "pre-main-reconciler-cli-integration-kyma-latest-k3d"
+                  # following regexp won't start build if only Markdown files were changed
+                  run_if_changed: "^((cmd\\S+|configs\\S+|internal\\S+|pkg\\S+)(\\.[^.][^.][^.]+$|\\.[^.][^dD]$|\\.[^mM][^.]$|\\.[^.]$|/[^.]+$))"
+                  labels:
+                    preset-build-pr: "true"
+                    preset-kyma-guard-bot-github-token: "true"
+                    preset-sa-vm-kyma-integration: "true"
+                    preset-gc-project-env: "true"
+                  annotations:
+                    pipeline.platform: k3d
+                  env:
+                    KYMA_TEST_SOURCE: "latest-release"
```

Comment:
Where is thing variable being used ? 🤔 I see that setting the version works (in the logs). Can you explain it ?

Reply:
Its used [here](https://github.com/kyma-project/test-infra/blob/92a1fe680720546d298182b52526dd550f8b12c0/prow/scripts/provision-vm-and-test-reconciler-with-cli-k3d.sh#L124)!